digital_comms/fixed_network/interventions.py

```diff
@@ -7,16 +7,16 @@
 ################################################################
 
 
-def get_all_distributions_ranked(system, ranking_variable, technology, reverse_value):
```

Comment:
On my branch I have updated the model to work in an asset neutral way, so you can just give it either exchanges, cabinets or distribution points. Therefore I'm not going to accept these proposed changes to interventions.py, as I have new updates that I will make to the master in the next few days which meet and hopefully exceed this integration requirement.